website/docs/r/ruleset.html.markdown

```diff
@@ -121,6 +148,15 @@ The following arguments are supported:
 * `uri` - (Optional) List of URI properties to configure for the ruleset rule when performing URL rewrite transformations (refer to the [nested schema](#nestedblock--action-parameters-uri)).
 * `version` - (Optional)
 
+<a id="nestedblock--ratelimiting-parameters"></a>
+**Nested schema for `ratelimit`**
+
+* `characteristics` - (Optional) List of parameters that define how Cloudflare tracks the request rate for this rule.
+* `period` - (Optional) The period of time to consider (in seconds) when evaluating the request rate.
+* `requests_per_period` - (Optional) The number of requests over the period of time that will trigger the Rate Limiting rule.
```

Comment:
```suggestion
* `requests_per_period` - (Optional) The number of requests over the period of time that will trigger the rate limiting rule.
```

Reply:
FYI, these are straight out of the dev docs if we need consistency.

Reply:
You're right! I thought these had been changed in the dashboard to lower case in the meanwhile, but they weren't. Let's keep the original case, then (`Rate Limiting rule`).

Reply:
And sorry for the mishap.